src/Umbraco.Web.UI.Client/src/less/pages/login.less

```diff
@@ -13,7 +13,9 @@
     left: 0;
     margin: 0 !important;
     padding: 0;
+    border: none;
     border-radius: 0;
+    overflow-y: auto;
```

Comment:
It actually inherited a border, which made it scroll 1px vertically and horizontally, when I tested it on my tablet and phone.